src/OpenTelemetry/Metrics/MetricPointStatus.cs

```diff
@@ -0,0 +1,50 @@
+// <copyright file="MetricPointStatus.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics
+{
+    internal enum MetricPointStatus
+    {
+        /// <summary>
+        /// Uninitialized <see cref="MetricPoint"/>.
+        /// </summary>
+        Unset,
+
+        /// <summary>
+        /// UpdatePending <see cref="MetricPoint"/>.
+        /// </summary>
+        UpdatePending,
```

Comment:
Please add a comment to indicate that the point is just created, and hasn't received the very 1st update.

Reply:
I think this status may be unnecessary, I'm attempting to remove it. If it ends up staying I'll update the comment.